salt/modules/mssql.py

```diff
@@ -58,7 +58,26 @@ def __virtual__():
     )
 
 
+def _get_db_args(**kwargs):
```

Comment:
Oops. Sorry I've missed this point. You don't need this function. You can use with as the following:
```python
with _get_connection(**kwargs) as conn:
```